src/Workspaces/Core/Desktop/Workspace/SQLite/SQLite.cs

```diff
@@ -0,0 +1,4003 @@
+﻿//
```

Comment:
The intent is not to check this in.  This is solely so i can get something real up and running.